src/components/modules/banners/contributions/ContributionsBannerWrapper.tsx

```diff
@@ -37,7 +37,7 @@ const withBannerData = (
         onClose,
         content,
         countryCode,
-        numArticles,
+        numArticles = 0, // TODO - does this work?
```

Comment:
I'm pretty sure it does! If 'work' means assigns to 0 if it's undefined in bannerProps